src/components/InvestigationsCard/useStyles.js

```diff
@@ -28,10 +28,16 @@ const useStyles = makeStyles(({ breakpoints, typography }) => ({
   media: {
     height: typography.pxToRem(322),
     width: typography.pxToRem(228),
+    "& span": {
+      boxShadow: `0 4px 8px 0 rgba(0,0,0,0.4)`,
+    },
   },
   featured: {
     height: `${typography.pxToRem(421)} !important`,
     width: `${typography.pxToRem(300)} !important`,
+    "& span": {
+      boxShadow: `0 4px 8px 0 rgba(0,0,0,0.4)`,
+    },
```

Comment:
Are the boxShadow changes meant to be on this PR? @nerdlet 😅

Reply:
@gertie-sheshe Yikes, pushed this in the wrong branch. i'll revert it